cudax/cmake/cudaxBuildCompilerTargets.cmake

```diff
@@ -29,7 +29,7 @@ function(cudax_build_compiler_targets)
     # __cplusplus to be defined for this to work:
     append_option_if_available("/Zc:__cplusplus" cxx_compile_options)
 
-    # cudax requires __VA_OPT__ for its unit tests
+    # We require the new preprocessor
```

Comment:
We can remove this entirely, since this is now part of the main cccl compiler interface. That gets inherited by all projects.